api/v1beta1/cloudstackmachine_webhook.go

```diff
@@ -32,6 +32,7 @@ import (
 // log is for logging in this package.
 var cloudstackmachinelog = logf.Log.WithName("cloudstackmachine-resource")
 
+// SetupWebhookWithManager creates a new webhook managed by passed manager
```

Comment:
Can you please add periods to the end of these comments? I'd like us to stick to the prevailing conventions.